jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt

```diff
@@ -134,8 +136,13 @@ enum class SourceOfEntry {
     }
 }
 
+interface AuthenticationDialog {
+    fun attempts(): Int
+    fun authType(): CredentialSourceId
+}
```

Comment:
```suggestion
interface AuthenticationDialog {
    val attempts: Int
    val authType: CredentialSourceId
}

interface Implementee : AuthenticationDialog {
    override var attempts: Int
        private set
    override var authType: CredentialSourceId
        private set
}
```
is slightly cleaner

Reply:
The suggestion adds an additional interface and takes us from 4 to 10 LoC. I would prefer to keep it as-is.

Reply:
s/interface Implementee/class Implementee/

it's the same interface you defined